### DISCUSSION TITLE
How to Start the gallery with only one preview picture?

### DISCUSSION THREAD

**Discussion by visaltgl** [fancyapps/ui#23] [Q&A]
I'm trying to achieve something like [this](https://stackoverflow.com/questions/21689188/multiple-fancybox-galleries-from-one-page-of-thumbnails). How to do this in v4?

In my case the trigger is a small image, but I don't want to show it in the slideshow / gallery.

I have tried using 'dialog' and 'gallery'.. but the images are becoming stacked instead of a slideshow.

**Comment by visaltgl** [comment]
Figured it out. Thanks for the awesome plugin.

**Reply by fancyapps** [reply]
Hi,

Thanks, the easiest way is to show one link and hide the rest, see this demo - https://gist.lipsum.app/gists/AZ

Also, you could create custom click event to your preview picture and start Fancybox manually, see this demo - https://fancyapps.com/playground/B0

Or you could use `data-fancybox-trigger` feature, see https://gist.lipsum.app/gists/B1
